include/opds_dumper.h

```diff
@@ -59,9 +59,10 @@ class OPDSDumper
    *
    * @param bookIds the ids of the books to include in the feed
    * @param query the query used to obtain the list of book ids
+   * @param partial whether the feed should include partial or complete entries
    * @return The OPDS feed.
    */
-  std::string dumpOPDSFeedV2(const std::vector<std::string>& bookIds, const std::string& query) const;
+  std::string dumpOPDSFeedV2(const std::vector<std::string>& bookIds, const std::string& query, bool partial = false) const;
```

Comment:
I would prefer to not have a default value for `partial`.

Partial/complete feed are somehow at equal level. Complete feed is not really the default/normal way to dump feed.
And as we use dumpOPDSFeedV2 only twice, there is no real reason to keep some kind of compatibility.
This is indeed a API break, but this release will have API break anyway.

Reply:
Done